diff --git a/node_modules/@capacitor-community/text-to-speech/Package.swift b/node_modules/@capacitor-community/text-to-speech/Package.swift
index d0d24fb..31a4c28 100644
--- a/node_modules/@capacitor-community/text-to-speech/Package.swift
+++ b/node_modules/@capacitor-community/text-to-speech/Package.swift
@@ -10,7 +10,7 @@ let package = Package(
             targets: ["TextToSpeechPlugin"])
     ],
     dependencies: [
-        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", from: "7.0.0")
+        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", "7.0.0"..<"9.0.0")
     ],
     targets: [
         .target(
